userapi/src/main/java/fk/prof/userapi/UserapiManager.java

```diff
@@ -52,7 +48,7 @@ public UserapiManager(UserapiConfigManager userapiConfigManager) throws Exceptio
     UserapiConfigManager.setDefaultSystemProperties();
     this.userapiConfigManager = Preconditions.checkNotNull(userapiConfigManager);
 
-    VertxOptions vertxOptions = new VertxOptions(userapiConfigManager.getVertxConfig());
+    VertxOptions vertxOptions = new VertxOptions();
```

Comment:
Lets still pass vertx config as mentioned above, can be an empty object for the time being